pkgs/applications/misc/echoip/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoPackage, fetchFromGitHub }:
+
+buildGoPackage rec {
+  name = "echoip-${version}";
+  version = "20181120-${lib.strings.substring 0 7 rev}";
```

Comment:
This should be  `unstable-2018-11-20`

> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". 

##### https://nixos.org/nixpkgs/manual/#sec-package-naming